internal/scorecard/config.go

```diff
@@ -50,7 +49,7 @@ func (e externalPluginEnv) String() string {
 }
 
 // validateConfig takes a config for a plugin and returns a nil error if valid or an error explaining why the config is invalid
-func validateConfig(config PluginConfig, idx int, version string) error {
+func (s Config) ValidateConfig(config PluginConfig, idx int) error {
```

Comment:
We don't use `s` in this function right? Can we unexport this function and move it to be a method on `PluginConfig` instead of `Config`?

Reply:
moved to PluginConfig, but left exported since its called from the cmd package.

Reply:
Can all of plugin config be moved to the `cmd` side?